### DISCUSSION TITLE
Error 500 on Asset Report by Owner

### DISCUSSION THREAD

**Discussion by echeng-aetheros** [opendcim/openDCIM#1427] [Q&A] [answered]
Hi, when I tried to click on this , it throws me Error 500
report_contact.php

anyone knows if I'm missing something?

Thanks

**Comment by samilliken** [comment]
Yes, you are missing something, but we can't say specifically because you didn't provide the error.log from apache or any details about your OS.

**Comment by echeng-aetheros** [comment]
When I click on the link, and did a tail -f on error log it shows nothing

but in access log it shows up this line
10.128.128.33 - echeng [25/Aug/2023:11:35:53 -0700] "GET /report_contact.php HTTP/1.1" 500 - "https://opendcim.xxxx.com/reports.php" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"

**Reply by samilliken** [reply]
Well you are missing something.   Look at the wiki for a list of the modules required and see which one(s) you may have left out.    Since you still didn't provide any info about your OS we can't be any more specific.

**Comment by echeng-aetheros** [comment]
I followed the guide on wiki
I'm on RockyOS 9.2
These are the php modules, I pretty much follow the guide on wiki

[root@opendcim httpd]# rpm -qa|grep php
php-common-8.0.27-1.el9_1.x86_64
php-pdo-8.0.27-1.el9_1.x86_64
php-xml-8.0.27-1.el9_1.x86_64
php-opcache-8.0.27-1.el9_1.x86_64
php-cli-8.0.27-1.el9_1.x86_64
php-mbstring-8.0.27-1.el9_1.x86_64
php-fpm-8.0.27-1.el9_1.x86_64
php-gd-8.0.27-1.el9_1.x86_64
php-8.0.27-1.el9_1.x86_64
php-pecl-zip-1.19.2-6.el9.x86_64
php-snmp-8.0.27-1.el9_1.x86_64
php-mysqlnd-8.0.27-1.el9_1.x86_64
php-intl-8.0.27-1.el9_1.x86_64
php-ldap-8.0.27-1.el9_1.x86_64

**Reply by wilpig** [reply, marked as answer]
You pretty much followed?   Yeah there's your problem.

https://github.com/opendcim/openDCIM/wiki/RHEL#rhel-9-installation-instructions 

I have once again verified this step by step to ensure that it is correct.  I did a minimal install with Rocky 9.2 and the wiki install came off perfectly and the asset report works fine.

**Reply by wilpig** [reply]
<img width="1545" alt="Screenshot 2023-08-26 at 11 26 30 PM" src="https://github.com/opendcim/openDCIM/assets/1653882/d3dad0ea-99b6-43ed-9b5a-3383f319dce8">

**Reply by wilpig** [reply]
<img width="1411" alt="Screenshot 2023-08-26 at 11 27 58 PM" src="https://github.com/opendcim/openDCIM/assets/1653882/22ee133f-9bc6-429c-b2dd-92f3b71f3c0a">

**Comment by echeng-aetheros** [comment]
![requirement](https://github.com/opendcim/openDCIM/assets/143215579/4652ef90-711a-4cf9-8aeb-4eecf6f31a03)
